src/main/java/Duke.java

```diff
@@ -1,10 +1,150 @@
+package ip.src.main.java;
+import java.util.Objects;
+import java.util.Scanner;
+import java.util.Arrays;
+import java.util.ArrayList;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private static String file = "ip/src/main/java/duke.txt";
+
+    private static ArrayList<String> letter = new ArrayList<>();
+    private static ArrayList<String> done = new ArrayList<>();
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    private static int i=0;
+
+    private static void printFileContents(String filePath) throws FileNotFoundException {
+        File f = new File(filePath); // create a File for the given file path
```

Comment:
In this case, the code is relatively self-explanatory. Consider removing the comments.